README_pt-BR.md

```diff
@@ -64,25 +64,29 @@ Toda a sintaxe pode ser encontrada [aqui](https://daringfireball.net/projects/ma
 | Links inline 	| \[texto\](url aqui) 	| Um [link](http://www.github.com) 	|
 | Imagens 	| \![Legenda\](url da img) 	| Uma imagem ![image](http://i.imgur.com/hRLuez2.png) 	|
 | Link+imagens 	| \[\![Legenda\](url da img)\](url para a pagina)\] 	| Me clique [![me](http://i.imgur.com/hRLuez2.png)](https://www.youtube.com) 	|
-| Notas de rodapé  	| Eu tenho mais \[^1\] pra dizer.   \[^1\]: diga aqui. 	| <a href="#section1">Ei, por favor leia as notas sobre essa tabela.  	|
 | Quebras de linha 	| Double space + enter 	|  	|
-| Listas não ordenadas 	| \* Item1     \*Item 2 	| <ul><li>item1</li><li>item2</li><li>item3</li><li>item4</li></ul> 	|
-| Listas Ordenadas  	| 1. Item a    2. Item b 	| <ol><li>itema</li><li>itemb</li><li>itemc</li><li>itemd</li></ol>  	|
-| Listas mistas 	| 1. Item 1      * item 1a 	|  <ol><li>itema</li></ol><ul><li> item1</li></ul>	|
+| Listas não ordenadas 	| \* Item1     \*Item 2 	| <ul><li>item1</li><li>item2</li> 	|
+| Listas Ordenadas  	| 1. Item a    2. Item b 	| <ol><li>itema</li><li>itemb</li>  	|
```

Comment:
I think it's better to keep a pattern. Since the previous items start with a lowercase letter, what do you think about keeping this here?